CRADLE/CallPeak/callPeak.py

```diff
@@ -168,11 +169,50 @@ def mergePeaks(peakResult):
 
 	for i in range(vari.CTRLBW_NUM):
 		ctrlBW[i].close()
+
+		if vari.I_LOG2FC:
+			normCtrlBW[i].close()
+
 	for i in range(vari.EXPBW_NUM):
 		expBW[i].close()
 
+		if vari.I_LOG2FC:
+			normExpBW[i].close()
+
 	return mergedPeak
 
+def updatePvalue(pvalues):
+	minPValue = np.min(pvalues)
+	
+	if minPValue == 0:
+		pvalue = np.nan
+	else:
+		pvalue = np.round((-1) * np.log10(minPValue), 2)
+
+	return pvalue
+
+def updateQvalue(qvalues):
+	minQValue = np.min(qvalues)
+
+	if minQValue == 0:
+		qvalue = np.nan
+	else:
+		qvalue = np.round((-1) * np.log10(minQValue), 2)
+	
+	return qvalue
```

Comment:
updatePvalue and updateQvalue are the same function, but with different parameter names. I think you should just have one function "newFC" (or something like that) that calculates the log2 fold change.
```suggestion
def newFC(values):
	minValue = np.min(values)
	
	return minValue == 0 ? np.nan : np.round((-1) * np.log10(minValue), 2)
```

Reply:
i agree. I think we just need to change the name of the function to `takingLog` because this function has nothing to do with FC (fold change)

Reply:
```suggestion
def takeMinusLog(values):
	minValue = np.min(values)
	
	return minValue == 0 ? np.nan : np.round((-1) * np.log10(minValue), 2)
```